plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -322,6 +322,12 @@ struct MutationScore {
     NamedType!(double, Tag!"MutationScore", 0.0, TagStringable) score;
 }
 
+struct FileScore {
+    SysTime timeStamp;
+    NamedType!(double, Tag!"MutationScore", 0.0, TagStringable) score;
```

Comment:
tag should be `MutationFileScore`, then we get a compilation error if they are accidentally mixed with `MutationScore`